precompile/deployerallowlist/config_test.go

```diff
@@ -0,0 +1,95 @@
+// (c) 2019-2020, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package deployerallowlist
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/ava-labs/subnet-evm/precompile"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/stretchr/testify/require"
+)
+
+func TestVerifyContractDeployerConfig(t *testing.T) {
+	admins := []common.Address{{1}}
+	tests := []struct {
+		name          string
+		config        precompile.StatefulPrecompileConfig
+		expectedError string
+	}{
```

Comment:
Should we create a standard allow list test suite that can be shared across all of the precompiles that use it for both `Configure` and `Equal` ?

Reply:
I tried to find a simple way but we need to give concrete struct as the input to the suit. I prefer unit tests to be dumb as possible.